### PR TITLE
Added Gourav Gosain to contributors.md

### PR DESCRIPTION
**Pull request by gosaingourav17**

### CODE REVIEW

**Comment by gosaingourav17** [comment]
For Hacktoberfest

**Comment by arshadkazmi42** [comment]
@gosaingourav17 looks good, tag the issue number in your PR description

**Comment by arshadkazmi42** [comment]
@gosaingourav17  why you have closed this? its not merged yet. you just have to add the issue id in your comment

**Comment by gosaingourav17** [comment]
I did a PR again

**Comment by arshadkazmi42** [comment]
@gosaingourav17 You have still not added the issue number here. Just add a comment with the issue number `#8`

File: Contributors.md [hunk 1]
```diff
@@ -1,14 +1,25 @@
 # Contributors
-
+- [Antonio Neto](https://github.com/aacgn)
```
Comment (arshadkazmi42) [inline Contributors.md]: Why is it showing alot of changes in your PR.
You should add only your name.

**Pull request closed without merging**